### PR TITLE
On-prem docs tweaks

### PR DESCRIPTION
**Pull request by davetroiano**
### Description 

On-prem tutorial docs tweaks and formatting fixes

### Author Validation

Built and sanity checked docs locally

- [x] Documentation: confluentinc/docs-platform#530

### CODE REVIEW

**Comment by ybyzek** [comment]
@davetroiano staged PR at https://github.com/confluentinc/docs-platform/pull/530 

Currently waiting for it to build, will review this PR once that one completes.

File: docs/on-prem.rst [hunk 1]
```diff
@@ -76,7 +76,7 @@ You can run it with optional settings:
 
       CLEAN=true ./scripts/start.sh
 
-#. ``cp-demo`` supports access to the |c3| GUI via either ``http://`` (the default) or secure ``https://``, the latter employing a self-signed CA and certificates generated during deployment. Due to |c3| integrations to other components include ksqlDB server, only one mode at a time is fully supported. To run ``cp-demo`` in ``https`` mode, set ``C3_KSQLDB_HTTPS=true`` when starting ``cp-demo``:
+#. ``cp-demo`` supports access to the |c3| GUI via either ``http://`` (the default) or secure ``https://``, the latter employing a self-signed CA and certificates generated during deployment. Because |c3| integrates with ksqlDB server, only one mode at a time is `supported <https://docs.confluent.io/platform/current/ksqldb/integrate-ksql-with-confluent-control-center.html#configuration-settings-for-ksqldb-and-c3-short>`__. To run ``cp-demo`` in ``https`` mode, set ``C3_KSQLDB_HTTPS=true`` when starting ``cp-demo``:
```
Comment (ybyzek) [inline docs/on-prem.rst:79]: The caveat is more subtle:

> To run ksqlDB queries from Control Center, both ksqlDB and Control Center must be running on either HTTP or HTTPS.

IIUC mixed mode works and is supported, but *to run queries* it has to be single mode.  Can you tweak this language a bit to more accurately reflect this?  Or...make it more vague and point to the docs for details ;)

**Review by ybyzek — CHANGES_REQUESTED** [review]
@davetroiano thanks for the PR 🎉  , please see comment inline.

File: docs/on-prem.rst (hunk 1 above)
Comment (davetroiano) [reply docs/on-prem.rst]: I tweaked to clarify this

Comment (ybyzek) [reply docs/on-prem.rst]: Thanks!

**Review by ybyzek — APPROVED** [review]
@davetroiano LGTM thank you for the PR 🎉

**Pull request merged**